yellowbrick/classifier/classification_report.py

```diff
@@ -122,17 +136,17 @@ def draw(self):
         Renders the classification report across each axis.
         """
         # Create display grid
-        cr_display = np.zeros((len(self.classes_), 4))
+        cr_display = np.zeros((len(self.classes_), len(self.measurements)))
 
 
         # For each class row, append columns for precision, recall, f1, and support
         for idx, cls in enumerate(self.classes_):
-            for jdx, metric in enumerate(('precision', 'recall', 'f1', 'support')):
+            for jdx, metric in enumerate(self.measurements):
```

Comment:
No change required, but by having `_displayed_scores` pre-filtered, this line of code will ensure `support` is displayed or not.